.github/workflows/github_release.yml

```
@@ -16,6 +17,8 @@ jobs:
           java-version: '17'
 
       - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)